filters/canada/british-columbia/vancouver/public-art-json-to-json.js

```diff
@@ -40,20 +42,48 @@ const filter = function (data, std_lib, schema, validator, stringify) {
             skip = true;
         }
         item.coordinates = coordinates;
-  
-        // add optional fields
-  
+
+        item.dates = create_dates_template();
+        add_if_not_null(item.dates.installed, "year", d.yearofinstallation);
+        remove_null_date_fields(item);
+        
+        add_if_not_null(item, "description", d.descriptionofwork);
+        
+        item.address = {}
+        add_if_not_null(item.address, "street_address", d.streetaddress);
+        item.address = remove_if_null(item.address);
```

Comment:
This line is redundant. You're already adding street_address only if it's not null